### DISCUSSION TITLE
Need help for starting contribution on MDAnalysis project

### DISCUSSION THREAD

**Discussion by nobody-butcat** [MDAnalysis/mdanalysis#4527] [Outreachy Discussions]
Hello, I need some help contributing to the said. I cannot understand where to check in order to make a contribution. can anyone please help?

**Comment by nobody-butcat** [comment]
I am a beginner at open source. so really some help would do!

**Reply by orbeckst** [reply]
In general you should get familiar with MDAnalysis (what does it do, who is using it, why does it exist...), and it's also a good idea to [install the MDAnalysis library](https://userguide.mdanalysis.org/stable/installation.html) and try it out by following the [Quickstart Guide](https://userguide.mdanalysis.org/stable/examples/quickstart.html). If you want to contribute through code, then we have a [Contribution Guide](https://userguide.mdanalysis.org/stable/contributing.html). You can also contribute in other ways (which we also highly appreciate), e.g., by helping with documentation, improving user experience across GitHub, https://www.mdanalysis.org/, and Discord, etc ...  

The **#outreachy channel in the MDAnalysis discord** (see [Participating](https://www.mdanalysis.org/#participating) for the invite link to discord) has been quite lively and many people interested in outreachy have had discussions there so I'd encourage you to look there.
